pkg/restapi/task.go

```diff
@@ -151,6 +151,9 @@ func (h *taskHandler) parseTask(r *http.Request) (*scheduler.Task, error) {
 	if err := render.DecodeJSON(r.Body, &t); err != nil {
 		return nil, err
 	}
+	if !t.Sched.StartDate.IsZero() && t.Sched.Cron.Spec != "" {
+		t.Sched.Cron.StartDate = t.Sched.StartDate
+	}
```

Comment:
Can there be a situation where `t.Sched.Cron.StartDate` is not zero value but gets overwritten? And why is it done here?

Reply:
Not really, seems it's a fix for a hidden bug, where adding the task didn't set correct StartDate in Cron :see_no_evil: This value will not be overriden, This value just must be copied from Sched.StartDate.